lib/lambda/cloudwatch-custom-widget/readers.mts

```diff
@@ -409,7 +409,7 @@ export async function getControlPanel(
     lambdaClient,
     input.resources.restatectlLambdaArn,
     "select * from nodes",
-  );
+  ) as unknown as Promise<NodeState[]>;
```

Comment:
We could enforce schema here in the future to break more explicitly if the shape ever changes but the JSON fields output by `restatectl` are derived from core Restate types so unlikely to randomly break.